src/EnergyPlus/InputProcessor.cc

```diff
@@ -3755,6 +3755,30 @@ namespace InputProcessor {
 
 	}
 
+	std::string
+	deAllCaps( std::string const & inString )
+	{
+		// Jason Glazer - August 2015
+		// turn all caps into caps only when first letter in word
+		std::string returnString;
+		bool wasSpace = true; //start out assuming a space so first character is in caps
+		for ( std::string::size_type i = 0; i != inString.size(); i++ ){
+			char curChar = inString[i];
+			if ( wasSpace ){
+				returnString.push_back( inString[i] );
+			}
+			else {
+				if ( curChar > 64 && curChar < 91 ){
+					curChar = curChar + 32;
+				}
+				returnString.push_back( curChar );
+			}
+			wasSpace = ( curChar == 32 );
+		}
+		return returnString;
+	}
+
+
```

Comment:
@JasonGlazer From the description "turn all caps into caps only when first letter in word" it's a little hard to tell what this is supposed to do.  Also, I'm curious how often this is called?  Just once on input?  Or routinely?  If routinely, I would guess that @mbadams5 or @DeadParrot would have a solution that didn't require push_back to build a new string each time.  If just once on input, it's likely not worth too much investigation.